dist/render.js

```diff
@@ -0,0 +1,44 @@
+class Renderer {
+    constructor() {
+        const source = $('#player-template').html();
+        const statsSource = $('#stats-template').html();
+        const myTeamsSource = $('#team-template').html();
+        this._template = Handlebars.compile(source);
+        this._statsTemplate = Handlebars.compile(statsSource);
+        this._myTeamsTemplate = Handlebars.compile(myTeamsSource);
+    }
+
+    showElement = element => element.css("visibility", "visible")
+    hideElement = element => element.css("visibility", "hidden")
+
+    _handleBarHelper = (elementToAppendTo, Template, data) => {
+        elementToAppendTo.empty();
+        let newHTML = Template(data);
+        elementToAppendTo.append(newHTML);
+    }
+
+    switchDisplay(element1, element2) {
+        element1.css("display", "none")
+        element2.css("display", "grid")
+    }
+
+    playerHandleBarHelper = (roster) => {
```

Comment:
Change the name of the functions with the name `Helper`. Just call them what they do...